recipes/pysam/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
 
 
 build:
-    number: 1
+    number: 2
```

Comment:
I don't think you need this version bump. You can even reset it to 0 as you will get a new build string by increasing the htslib package in the env_matrix.